src/core/toga/widgets/splitcontainer.py

```diff
@@ -11,7 +11,15 @@ class SplitContainer(Widget):
             If no style is provided then a new one will be created for the widget.
         direction: The direction for the container split,
             either `SplitContainer.HORIZONTAL` or `SplitContainer.VERTICAL`
-        content(``list`` of :class:`toga.Widget`): The list of components to be split.
+        content(``list`` of :class:`toga.Widget`): The list of components to be
+            split or tuples of components to be split and adjusting parameters
+            in the following order:
+        widget (:class:`toga.Widget`): The widget that will be added.
+        weight (float):
+        resize (boolean): Should the content expand when the widget is resized.
+        shrink (boolean): Can the content be made smaller than its requisition.
+        wide_handle (boolean): Does the seperation between the contents provide
+            stronger visual separation or notto be split.
```

Comment:
I don't think this needs to be an option. I'm OK with Toga being a little opinionated here, and picking either a wide or narrow style for split separators (and I think wide is almost always going to be preferable).